src/controller/securemgr/authenticator/authenticator.go

```diff
@@ -82,7 +82,7 @@ func Init(passPhraseJWTPath string) {
 		passphrase = []byte(randString(16))
 		err = ioutil.WriteFile(passPhraseJWTFilePath, passphrase, 0666)
 		if err != nil {
-			log.Println(logPrefix, "cannot create "+passPhraseJWTFilePath+": ", err)
+			log.Println(logPrefix, "Cannot create passPhraseJWT.txt:", err)
```

Comment:
Looks good, but ```'passPhraseJWT.txt'``` also looks like a filename( information).
What about you use just a simple text like ```'passphrase file'```?

Reply:
@mgjeong Thank you for your good question.
As I wrote earlier, this file should contain a phrase for generating a SHA-based JWT (we recommend using an RSA-based JWT). And this is described in the documentation, hence the attacker knows this.
If you look at the threat tree, then you should **divide the type of attack into remote and with direct access to the device**. At the current moment, the use of the token allows us to **protect against a remote attack**, but not against an attack when there is access to the device.
Of course, our goal is to provide maximum protection, and we will do this step by step.
In the future, I suppose we will store sensitive information in a secure storage.

Reply:
Okay. I got it.
Thanks.